src/borg/testsuite/archiver.py

```diff
@@ -1560,7 +1560,7 @@ def patched_fchown(fd, uid, gid):
             assert xattr.getxattr(b"input/file", b"security.capability") == capabilities
 
     @pytest.mark.skipif(
-        not xattr.XATTR_FAKEROOT, reason="xattr not supported on this system or on this version of" "fakeroot"
+        not xattr.XATTR_FAKEROOT, reason="xattr not supported on this system or on this version of " "fakeroot"
```

Comment:
hmm, if that's what black made out of this, guess we better merge the 2 strings into 1.

see also the other 2 places with same string.

Reply:
@KN4CK3R did you see my feedback?